packages/@okta/vuepress-site/docs/guides/oie-upgrade-mfa-enroll-policy/main/index.md

```diff
@@ -1,48 +1,52 @@
 ---
-title: Understand MFA enrollment policy API changes after the upgrade
+title: Understand authenticator enrollment policy API changes after the upgrade
 meta:
   - name: description
-    content: Learn how to use and manage the MFA enrollment policy with the API in Okta Identity Engine.
+    content: Learn how to use and manage the authenticator enrollment policies with the API in Okta Identity Engine.
 ---
 
 <ApiLifecycle access="ie" />
 
 ## Overview
 
-With Okta Identity Engine, the definition of factors and authenticators have been differentiated to align with industry standards. Identity Engine uses authenticators in its MFA enrollment policy settings, whereas Okta Classic Engine uses factors in its MFA enrollment policy settings.
+With Okta Identity Engine, the definition of factors and authenticators have been differentiated to align with industry standards. Identity Engine uses authenticators in its authenticator enrollment policy settings, whereas Okta Classic Engine uses factors in its Multifactor (MFA) enrollment policy settings.
 
-After upgrading your org to Identity Engine, new MFA enrollment policies created in the Admin Console are configured using authenticators. Existing MFA enrollment policies, created before upgrading to Identity Engine, are still configured with factors. However, if an existing MFA policy was modified and saved in the Admin Console, the factors in that policy are converted to authenticators. This conversion is seamless to admin users that are managing policies in the Admin Console.
+In Identity Engine, the MFA Enrollment Policy name has changed to [authenticator enrollment policy](/docs/reference/api/policy/#authenticator-enrollment-policy). Classic Engine still refers to the same policy as [Multifactor (MFA) Enrollment Policy](/docs/reference/api/policy/#multifactor-mfa-enrollment-policy). In the API, the policy type of `MFA_ENROLL` remains unchanged, however, the `settings` data contains authenticator or factors, depending on the configuration.
 
-For existing apps that manage and use MFA enrollment policies programmatically through the [Policy API](/docs/reference/api/policy/), some development work is required to handle the factor to authenticator conversion schema changes after the org is upgraded to Identity Engine. This guide provides you with key API considerations to upgrade your app for MFA enrollment flows in Identity Engine.
+After upgrading your org to Identity Engine, new authenticator enrollment policies created in the Admin Console are configured using authenticators. Authenticator enrollment policies created before upgrading to Identity Engine, are still configured with factors. However, if an existing policy was modified and saved in the Admin Console, the factors in that policy are converted to authenticators. This conversion is seamless to admin users that are managing policies in the Admin Console.
```

Comment:
Not sure if this is necessary to say, but if even if they're not modified, just saved, like the saved button was pressed then it will converted anyway.

Reply:
I think this would be useful information for the user. 
I'll change "..was modified and saved.." to "... was saved ..." add this note:
`**Note:** Regardless if you edit the authenticator enrollment policy, when you click the **Saved** button in the Admin Console, the factor-to-authenticator conversion occurs.  `